pjsip/include/pjsua2/endpoint.hpp

```diff
@@ -1162,6 +1162,14 @@ struct EpConfig : public PersistentObject
 /* This represents posted job */
 struct PendingJob
 {
+    PendingJob():autoDelete(true){};
+
+    /**
+     * If this is set to true, the job will be automatically be deleted
+     * after the job is executed.
+     */
+    bool    autoDelete;
```

Comment:
What if this is a private member, initialized via constructor param with the default is true (as existing)?